project/models.py

```diff
@@ -42,6 +42,6 @@ class List(db.Model):
 
     id = db.Column(db.Integer, primary_key = True, autoincrement = True)
     task = db.Column(db.String(140), nullable = False)
-    deadline = db.Column(db.DateTime, nullable = False)
+    deadline = db.Column(db.String(10), nullable = False)
```

Comment:
Me and @ranatlm19  had a discussion where he asked me to store deadline in String format of "dd-mm-yyyy"